shell/machine-config/vmwarevsphere.vue

```diff
@@ -1022,6 +1019,7 @@ export default {
               {{ t('cluster.machineConfig.vsphere.networks.label') }}
             </label>
             <ArrayListSelect
+              v-if="!!value.network"
```

Comment:
I also don't think that we want to be hiding our components if a value doesn't exist. Can we remove this for now and follow up in a separate issue/discussion to see if we want to implement this behavior?

Reply:
yes, I haven't updated this to what we discussed yet